src/coreclr/jit/codegenloongarch64.cpp

```diff
@@ -7068,7 +7117,18 @@ void CodeGen::genCall(GenTreeCall* call)
         }
         else if (abiInfo.IsSplit())
         {
-            NYI("unimplemented on LOONGARCH64 yet");
+            assert(compFeatureArgSplit());
+            genConsumeArgSplitStruct(argNode->AsPutArgSplit());
+
+            regNumber argReg   = (regNumber)((unsigned)abiInfo.GetRegNum() + idx);
+            regNumber allocReg = argNode->AsPutArgSplit()->GetRegNumByIdx(idx);
```

Comment:
It looks like the `idx` variable is not defined anywhere.

Reply:
Yes, Thanks, I will update it.